### ISSUE TITLE
Appending a Controller-based input in a useFieldArray has incorrect defaultValue

### ISSUE THREAD

**Issue by thomsocialbrothers** [react-hook-form/react-hook-form#10298]
### Version Number

6.15.8 (also replicated at 7.43.9)

### Codesandbox/Expo snack

https://codesandbox.io/s/quizzical-frost-mllvul?file=/src/App.tsx

### Steps to reproduce

1. Visit codesandbox
2. The correct defaultValues are shown
3. Click "Append"
4. The newly appended field has the value from `useForm()`'s `defaultValue`, instead of the value provided to `prepend()`

### Expected behaviour

That the newly appended field has the value provided in `prepend()`, and not the initial defaultValue of that index provided in `defaultValues`.

Also, this doesn't happen with regular input fields that use `register()`. 

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hekatx** [comment]
Couldn't replicate at 7.43.9. 

Working example at 7.43.9: https://codesandbox.io/s/inspiring-borg-7inod5?file=/src/App.tsx

**Comment by hekatx** [comment]
> @hekatx @bluebill1049 Thank you for your codesandbox. However, in that example both the defaultValue and the value that is passed to `prepend()` are not shown, which are the cause of this issue. Am I missing something in the migration guide? As far as I can see the API for these functionalities has not changed.

Yes, the render method props (`value` and `onChange`) of Controller have been removed, use `field` instead.

https://react-hook-form.com/api/usecontroller/controller/
